### ISSUE TITLE
outline appears on some texts or paragraphs when changing pages

### ISSUE THREAD

**Issue by illusionaz** [edrlab/thorium-reader#2084]
When changing pages a thin yellow line appears on paragraphs and sometimes on words, this is distracting. Is there a way I can disable this? Thank you
Images for reference
![example2](https://github.com/edrlab/thorium-reader/assets/105545125/e5ea29e0-f8a6-48b2-8218-5439ecc0b56d)
![example1](https://github.com/edrlab/thorium-reader/assets/105545125/b4bae8a8-b367-4669-8913-2705c3f89549)

**Comment by segln** [comment]
It doesn't seem like this issue has been resolved yet, so I think it should remain open.
Do you have plans to create an option to disable this feature?

**Comment by mghenderson** [comment]
I'm experiencing this "visual distraction" at least once per page turn.  I'm not a JS guy, so when I tried to follow it through the code base I ended up in the [r2-navigator-js](https://github.com/readium/r2-navigator-js) library, but even if that's correct I have no idea how to disable it.  Are there any plans to address this issue? I'd love to continue to use thorium but I can't handle the visual distraction.
